OmniBLE/Bluetooth/PeripheralManager+Omnipod.swift

```diff
@@ -168,7 +168,7 @@ extension PeripheralManager {
         
         log.debug("Writing Data Value %@", value.hexadecimalString)
         
-        try writeValue(value, for: characteristic, type: .withResponse, timeout: timeout)
+        try writeValue(value, for: characteristic, type: .withoutResponse, timeout: timeout)
```

Comment:
This was the primary fix.

Reply:
Just a caution that I did the same for a playground that sent these commands. Doing this seemed to get the commands further along but I only think because it didn't validate them but actually had failed on a prior command. I'm not 100% sure on it but that is what I recall from experimentation with this arg.

Reply:
Reverted.